src/pageComponents/infoPage/GuardianSection.tsx

```diff
@@ -0,0 +1,51 @@
+'use client'
+
+import { HulenPortableText } from '@/components/GenericPageContent'
+import { SanityImageComponent } from '@/components/sanity'
+import { DEFAULT_LAYOUT_MAXWIDTH } from '@/configs/constants'
+import type { GuardianInfoPageContent } from '@/types/sanity/infoPages/guardianInfoPage'
+import { useLanguage } from '@/util/LanguageContext/LanguageContext'
+import { Box, Stack, Typography } from '@mui/material'
+
+export const GuardianSection = ({ content }: { content: GuardianInfoPageContent }) => {
+  const { language } = useLanguage()
+
+  return (
+    <Stack sx={{ alignItems: 'center', padding: '2rem', width: '100%' }}>
+      <Stack
+        sx={{
+          width: '100%',
+          maxWidth: DEFAULT_LAYOUT_MAXWIDTH,
+          gap: '2rem',
+          display: 'flex',
+          flexDirection: 'column',
+          alignItems: 'center',
+        }}
+      >
+        <Box sx={{ width: '100%', alignSelf: 'start', textAlign: 'center' }}>
+          <Typography variant='h1' fontWeight={700} width={'100%'}>
+            {content.header[language]}
+          </Typography>
+          <HulenPortableText genericSanityPageProps={content.intro} />
+
+          <Stack sx={{ flexDirection: { xs: 'column', sm: 'row' }, paddingTop: '2rem' }} gap={5}>
```

Comment:
Spesifikk liten MUI best practice.

Stack er jo bare en div kontainer med display='flex' og flexDirection='column'.
Den eksisterer for å mentalt bryte litt opp i det som fort blir BOX BOX BOX BOX BOX nedover treet, så når man ser en stack så skal man tenke "yeah denne går nedover".

Men når du conditionally sier at den går column eller row basert på skjermstørrelse så kan det bryte litt med heurestikken, så ville heller brukt Box taggen så man ikke anntar at det alltid er en column. 

also gap er magic number her, skriv 2.5rem istedenfor